plugins/pulp_python/plugins/migrations/0002_smaller_model_for_json.py

```diff
@@ -0,0 +1,20 @@
+"""
+This migration removes `home_page`, `platform`, and `license` from the units_python_package
+collection. Additionally, it renames `_filename` to `filename` and makes it the pk.
```

Comment:
It eventually clicked that pk was primary key, but I'd prefer it to be spelled out for me.